wagtail/images/templates/wagtailimages/edit_handlers/multiple_images_panel.js

```diff
@@ -0,0 +1,192 @@
+{% load l10n %}
```

Comment:
Same comment here with duplication with [`wagtail/images/static_src/wagtailimages/js/add-multiple.js`](https://github.com/wagtail/wagtail/blob/c7a418b1c360204ab8393386a2dd72312e21993a/wagtail/images/static_src/wagtailimages/js/add-multiple.js). Is there a way to consolidate/refactor to eliminate duplication?